client/client.go

```diff
@@ -25,8 +23,8 @@ type Config struct {
 // ProxyClient is an interface for communicating with the EigenDA proxy server
 type ProxyClient interface {
 	Health() error
-	GetData(ctx context.Context, cert *common.Certificate, domain common.DomainType) ([]byte, error)
-	SetData(ctx context.Context, b []byte) (*common.Certificate, error)
+	GetData(ctx context.Context, cert []byte, domain server.DomainType, commitmentType server.CommitmentType) ([]byte, error)
+	SetData(ctx context.Context, b []byte, commitmentType server.CommitmentType) ([]byte, error)
```

Comment:
knit - commitmentType could be better encapsulated under a reduced rep like `ct`

Reply:
wdym? shorter variable name?